### ISSUE TITLE
Accessibility Checker should fail Github check if issues are found

### ISSUE THREAD

**Issue by mcritchlow** [ucsdlib/dmr#84]
Currently the accessibility checker runs correctly via CircleCI and reports found issues. It should, however, probably fail if it finds such issues. Please determine whether this is possible.

**Comment by mcritchlow** [comment]
@VivianChu - I believe we decided that this was not desirable, since the developers of this plugin see the accessibility report not failing as a feature. Should we close this out?

**Comment by VivianChu** [comment]
@mcritchlow I will close this out.  Thanks
